index.js

```diff
@@ -37,14 +39,30 @@ fetch(FORUM_API)
     isLoading = false;
   });
 
-const displayPostList = (postList) => {
+const displayPostList = (postList, users) => {
   postList
     .filter((post) => post["category_id"] in supportedTopicCategories)
-    .forEach(displayPost);
+    .forEach((post) => displayPost(post, users));
 };
 
-const displayPost = (post) => {
+const displayPost = (post, users) => {
   const category = supportedTopicCategories[post["category_id"]];
+  const posters = post.posters.map(({ user_id: userId }) => userId);
+
+  let postersAvatars = "";
+  const displayPosterAvatar = (posterId) => {
+    const poster = users.find((user) => user.id == posterId);
+    const avatarTemplate = poster["avatar_template"].replace("{size}", "25");
+    const posterAvatar = avatarTemplate.startsWith("/")
+      ? `${FORUM_AVATARS}/${avatarTemplate}`
+      : avatarTemplate;
+    postersAvatars += `
+      <a href="${FORUM_USER}/${poster.username}" target="_blank">
+        <img src="${posterAvatar}" title="${poster.username}"/>
+      </a>
```

Comment:
Hey,

- every `<img>` element MUST have an alternative text - `alt` attribute ( I know there are missing alts for other images, need to be resolved in another issue I guess) [[Learn more about alternative text](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Images_in_HTML#alternative_text)]
- it is a good practice to set `width` and `height` attributes [[Learn about width and height att](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Images_in_HTML#width_and_height)] so the browser can reserve the space for the image before it's loaded and does not cause layout shift [[Learn about layout shift](https://medium.com/nerd-for-tech/images-without-dimensions-layout-shift-47b26e01ba62)]
You have probably seen layout shift before when you were loading a page and things start to move around as images and other elements finished downloading - most common example are ad banners

Read a little bit about it and try to resolve those issues 😄

Reply:
Will do, thank you!

Reply:
When you think about what to include in the `alt` attribute, imagine that the image does not load for whatever reason or you are a screen reader user. The image in this context does not convey any meaning, it is decorative, so it could be left out empty `alt=""` but it functions as a link so the best approach to this is give a simple description what it does and where will the link take you.
My suggestion: `alt="Open [nickname]'s profile"`
My source of information: https://www.w3.org/WAI/tutorials/images/

Reply:
Oh, alright.